### DISCUSSION TITLE
Avoid conflicts with LL2.10?

### DISCUSSION THREAD

**Discussion by izywizy** [NightscoutFoundation/xDrip#2951] [General]
As you all probably know LL2.10 has ben released and is now a CGM of sorts.

I start a new Libre2 on Sunday, I have the new LL app installed and I want to avoid any conflict with xdrip.

My normal process with earlier versions is to start the L2 with the phone app and once the L2 has warmed up setup xdrip then once calibrated and giving me readings I am able to scan with the LL app, should it be the same procedure with new app as I don't want LL grabing the blu-tooth connection?

Also how do I stop the 'sensor age has gone backwards' message with normally happens after a LL NFC scan?

**Comment by PhilTU** [comment]
There is no additional conflict with LL 2.10 as the alarms (if enabled) already set up the BLE link and the changes to the app are mainly to display the BLE glucose data on the screen.

I would advise leaving all alarms off in LL and if you can deny it Bluetooth / Location / Nearby devices permissions. In this state it does not establish a Bluetooth connection and potential interaction is minimised. 

The sensor age thing I don't have a solution, it seems to arise if you NFC scan very close to a 1 minute value transmission, so you may be able to look for a data point arriving in xDrip+ and scan after that.

**Reply by izywizy** [reply]
Thanks, only permission I can block with this version is nearby devices, location isn't listed as a permission at all.

**Reply by PhilTU** [reply]
that varies with Android version , at one point Bluetooth fell into "Location services". Recently the Librelink app doesn't declare it uses that so you 
can't block it. Nearby devices is I think a term used in later versions of Android.

Either way, no Librelink = No conflict. Once a sensor is runing you can delete it.
